content/reusability.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+permalink: /reusability/
+title: Reusability  
+---
+
+## Reusability and Reproduciblity with CWL
+CWL workflows are portable, reusable, and flexible.  Reproduction, reuse, and replication of computational methods requires a complete description of what computer applications were used, how exactly they were used, and how they were connected to each other. To this end, CWL enables portability by being explicit about inputs/outputs to form the data flow, data locations and execution models. CWL supports using software container technologies, such as Docker and Singularity, to enable portability. For increased flexiblity, individual CWL tool definitions can be reused in any new workflow.  
```

Comment:
```suggestion
CWL workflows are portable, reusable, and flexible. Reproduction, reuse, and replication of computational methods require a complete description of what computer applications were used, how exactly they were used, and how they were connected to each other. To this end, CWL enables portability by being explicit about inputs/outputs to form the data flow, data locations and execution models. CWL supports using software container technologies, such as Docker and Singularity, to enable portability. For increased flexiblity, individual CWL tool definitions can be reused in any new workflow.  
```

Reply:
hmmm..`require` is one time, yeah? `requires` implies ongoing; the description must remain complete

Reply:
I thought it was "reproduction, reuse, and replications require a complete description..."; no?

Reply:
> > > I've got another CWL adopter to list
> > > ### NCI Genomic Data Commons
> > 
> > 
> > What section does this go in? Should I just put it below the NCI Cancer Genomics Cloud entry?
> 
> Sure
> 
> > Should I include their logo: https://gdc.cancer.gov/sites/all/themes/gdc_bootstrap/logo.png ?
> 
> I think so. We'll do a final logo check later.

I have removed this.  We can discuss later, but I wouldn't recommend adding it.